src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -498,12 +502,15 @@ def list_apps(client, app_id=None, display_name=None, identifier_uri=None, query
     if identifier_uri:
         sub_filters.append("identifierUris/any(s:s eq '{}')".format(identifier_uri))
 
-    if not sub_filters:
-        logger.warning('In a future release, if no filter arguments are provided, CLI will output only the first'
-                       ' 100 objects to minimize wait times. You can still use --all for the old behavior,'
-                       ' though it is not recommended')
-
-    return client.list(filter=(' and '.join(sub_filters)))
+    result = client.applications.list(filter=(' and '.join(sub_filters)))
```

Comment:
It looks like wait times are not affected by `include-all` since the slicing is done on the client-side?

Reply:
Yes, `--include-all` is the worst case which we don't really expect users to do, but it is there for whatever reasons users might still want to list the whole world.